app/services/exports/lettings_log_export_service.rb

```diff
@@ -91,6 +93,11 @@ def write_export_archive(export, lettings_logs)
 
       # Write all archives
       archive_datetimes = {}
+      @logger.info("Following archives to write:")
+      # rubocop:disable Style/CombinableLoops
+      lettings_logs_per_archive.each do |archive, lettings_logs_to_export|
```

Comment:
if we define each of these loops that rubocop wants to combine as a method, I think that might avoid us having to disable the linter

Reply:
actually I'm not sure that's true, but regardless do we really want to be logging before we're actually doing the thing we're logging to say we're doing?

Reply:
Yeah, I think cause it's running out of memory before completing it might not print all of these out if added to the loop below, I will likely remove this logging it's mostly for testing atm

Reply:
Sure, I'm convinced this is fine then :)